packages/common/src/hooks/purchaseContent/usePurchaseContentErrorMessage.ts

```diff
@@ -14,7 +15,11 @@ const messages = {
   minimumPurchase: (minAmount: number) =>
     `Total purchase amount must be at least $${formatPrice(minAmount)}.`,
   maximumPurchase: (maxAmount: number) =>
-    `Total purchase amount may not exceed $${formatPrice(maxAmount)}.`
+    `Total purchase amount may not exceed $${formatPrice(maxAmount)}.`,
+  badAmount: (minAmount: number, maxAmount: number) =>
+    `Total purchase amount must be between $${formatPrice(
+      minAmount
+    )} and ${formatPrice(maxAmount)}`
```

Comment:
changed the language slightly to avoid having to go back and create two separate error types, let me know if not ok @raymondjacobson

Reply:
sounds good!